OSDOCS-16947#CQA work Stor3 - CSI group snapshots (#114215)
openshift/openshift-docs: modules/persistent-storage-csi-group-snapshots-limitations.adoc

[id="persistent-storage-csi-group-snapshots-limitations_{context}"]
= CSI volume group snapshots limitations

[role="_abstract"]
Volume group snapshots have operational and consistency limitations that affect how you can use them for data protection and recovery. Understanding these constraints helps you design appropriate backup and recovery strategies for multi-volume applications.

> Volume group snapshots has the following limitations:

Reviewer: ```suggestion
A volume group snapshot has the following limitations:
```
basic grammar

> * Does not support reverting an existing persistent volume claim (PVC) to an earlier state represented by a snapshot It only supports provisioning a new volume from a snapshot.

Reviewer: ```suggestion
* Does not support reverting an existing persistent volume claim (PVC) to an earlier state represented by a snapshot. You can only provision a new volume from a snapshot.
```
the "It" statement just didn't make sense to me

> * No guarantees of application consistency, for example, crash consistency, are provided beyond those provided by the storage system. For more information about application consistency, see _Quiesce and Unquiesce Hooks_ under _Additional resources_.

Reviewer: ```suggestion
* No guarantees of application consistency, for example, crash consistency, are provided beyond those provided by the storage system. For more information about application consistency, see "Quiesce and Unquiesce Hooks" in the _Additional resources_ section.
```
no "under" (accessibility); and everywhere else I see quotes, not itals for titles
